packages/experiments/src/utilities/IHTMLSlots.ts

```diff
@@ -1,7 +1,13 @@
 import { ISlotProp } from './ISlots';
 
-// Generic slot definitions for intrinsic HTML elements
+/**
+ * Generic slot definition allowing common HTML attributes. Applicable for most intrinsic slots.
+ */
+export type IHTMLGenericSlot = ISlotProp<React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>, React.ReactNode>;
```

Comment:
It's funny because the word generic means something different in typescript / type systems. And actually THIS type should be called `IHTMLElementSlot` while the one below should actually be called `IHTMLGenericSlot<T>` or maybe better named `IHTMLElementSlot<T>`

Reply:
I had it as `IHTMLSlot`.. I'll change it back to that.